titus_isolate/isolate/workload_manager.py

```diff
@@ -13,7 +12,7 @@
 
 
 class WorkloadManager:
-    def __init__(self, cpu, cgroup_manager):
+    def __init__(self, cpu, cgroup_manager, allocator_class=IntegerProgramCpuAllocator):
```

Comment:
We'll need to make the caller of this configurable with fast properties at some point, but isn't needed in this PR.